pkgs/applications/office/gnucash/default.nix

```diff
@@ -1,106 +1,210 @@
+    make \
+      test-account-object \
+      test-address \
+      test-agedver \
+      test-app-utils \
+      test-aqb \
+      test-autoclear \
+      test-backend-dbi \
+      test-business \
+      test-column-types \
+      test-commodities \
+      test-customer \
+      test-dom-converters1 \
+      test-dynload \
+      test-employee \
+      test-engine \
+      test-exp-parser \
+      test-gnc-glib-utils \
+      test-gnc-guid \
+      test-gnc-int128 \
+      test-gnc-numeric \
+      test-gnc-path-util \
+      test-gnc-rational \
+      test-group-vs-book \
+      test-guid \
+      test-import-account-matcher \
+      test-import-backend \
+      test-import-map \
+      test-import-parse \
+      test-import-pending-matches \
+      test-incompatdep \
+      test-job \
+      test-kvp-frames \
+      test-kvp-value \
+      test-link-module-tax-us \
+      test-link-ofx \
+      test-load-backend \
+      test-load-c \
+      test-load-engine \
+      test-load-example-account \
+      test-load-xml2 \
+      test-lots \
+      test-modsysver \
+      test-numeric \
+      test-object \
+      test-print-parse-amount \
+      test-qof \
+      test-qofquerycore \
+      test-qofsession \
+      test-query \
+      test-querynew \
+      test-recurrence \
+      test-resolve-file-path \
+      test-scm-query \
+      test-scm-query-string \
+      test-split-register-copy-ops \
+      test-split-vs-account \
+      test-sqlbe \
+      test-string-converters \
+      test-sx \
+      test-tokenizer \
+      test-transaction-reversal \
+      test-transaction-voiding \
+      test-userdata-dir \
+      test-userdata-dir-invalid-home \
+      test-vendor \
+      test-xml-account \
+      test-xml-commodity \
+      test-xml-pricedb \
+      test-xml-transaction \
+      test-xml2-is-file
+
+      export LD_LIBRARY_PATH="$PWD/lib:$PWD/lib/gnucash:$PWD/lib/gnucash/test:$PWD/lib/gnucash/test/future"
   '';
-  doCheck = true;
 
-  meta = {
-    description = "Personal and small-business financial-accounting application";
+  postInstall = ''
+    rm $out/bin/gnc-fq-update     # remove GnuCash update utility
```

Comment:
Is it possible to tell the build system to not build it in the first place?

Reply:
I think I would have to patch it out of https://github.com/Gnucash/gnucash/blob/4.9/gnucash/CMakeLists.txt . That file is... _daunting_. I followed the existing pattern of just `rm`ing it for fear of falling down a rathole:

https://github.com/NixOS/nixpkgs/blob/37b0c5d39a87a231886effbde814257182b5be0c/pkgs/applications/office/gnucash/default.nix#L59-L64

Reply:
I think we can maybe just remove the two occurrences in https://github.com/Gnucash/gnucash/blob/4.9/libgnucash/quotes/CMakeLists.txt

Reply:
Thank you for the pointer. Fixed in dccda595ca631314b988a300e5a8058d444c8e38 .

I did the same for valgrind.